packages/playwright-test/src/plugins/vitePlugin.ts

```diff
@@ -314,11 +314,18 @@ function vitePlugin(registerSource: string, relativeTemplateDir: string, buildIn
       lines.push(registerSource);
 
       for (const [alias, value] of componentRegistry) {
-        const importPath = value.isModuleOrAlias ? value.importPath : './' + path.relative(folder, value.importPath).replace(/\\/g, '/');
-        if (value.importedName)
-          lines.push(`import { ${value.importedName} as ${alias} } from '${importPath}';`);
-        else
-          lines.push(`import ${alias} from '${importPath}';`);
+        const importPath = value.isModuleOrAlias
```

Comment:
Let's bring the formatting back to what it was so that it was easier to track what actually changed.

Reply:
Fixed this in: https://github.com/pastelsky/playwright/pull/2. @pastelsky could you take a look at this?